### PR TITLE
obsolete tip of digit

### LINKED ISSUE #1701: UBERON:7500041 tip of digit
**Uberon term**
UBERON:7500041 tip of digit

**Suggested revision and reasons**
Digits are called digits in both hand and feet (manus and pes more broadly), so why is UBERON:7500041 'tip of digit' ("The distal-most point of a digit.") equivalent to "anatomical point and distalmost_part_of some phalanx of pes"?
"tip of digit" is also an exact synonym of UBERON:0009551 'distal segment of digit'. Perhaps UBERON:7500041 tip of digit should be obsoleted?

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #1701

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224421,7 +224423,7 @@ name: line from the distal-most end of the longest phalax of the foot to the pos
 def: "An anatomical line of a hindfoot that connects the end of the longest digit to the posterior-most part of a foot." [https://orcid.org/0000-0003-2699-3066]
 intersection_of: UBERON:0006800 ! anatomical line
 intersection_of: connects UBERON:7500039 ! posterior of pes
-intersection_of: connects UBERON:7500041 ! tip of digit
+intersection_of: connects UBERON:7500041 ! obsolete tip of digit
```
Comment (cmungall) [inline src/ontology/uberon-edit.obo:224426]: why isn't this flagged automatically?

Comment (nicolevasilevsky) [reply src/ontology/uberon-edit.obo:224426]: I fixed this!

Comment (matentzn) [reply src/ontology/uberon-edit.obo:224426]: The check was on master but not yet on this branch - now it is, so will fail. @nicolevasilevsky this is indeed _not_ fixed at the moment!

Comment (matentzn) [reply src/ontology/uberon-edit.obo:224426]: Oh it was fixed..

**Review by matentzn — APPROVED** [review]

**Pull request merged**